roles/k8s_best_practices_certsuite/templates/certsuite_config.yml.j2

```
@@ -64,6 +64,19 @@ connectAPIConfig:
   proxyPort: ""
 {% endif %}
 
+## collector DB
+{% if kbpc_partner_name|length %}
+partnerName: {{ kbpc_partner_name }}
+{% else %}
+#partnerName: ""
+{% endif %}
```

nit: It's a bit strange leaving the comment when not defined, but in any case you could do this:
```suggestion
{% if kbpc_partner_name|length == 0 %}#{% endif %}partnerName: {{ kbpc_partner_name }}
```

---

yeah, it's just to leave the same commented file we have right now in case the variable is not provided. But your suggestion looks really nice, let me add it